Make UMD optional, due to https://phabricator.babeljs.io/T7084

jpnelson opened the pull request.

jpnelson committed:
Make UMD optional, due to https://phabricator.babeljs.io/T7084

joscha commented on lib/babel.js:
```diff
@@ -4,13 +4,19 @@ var report = require('./report');
+  );
+
+  if (opts.umd) {
+    plugins.push('babel-plugin-transform-es2015-modules-umd');
```
this one doesn't need to be resolved?

jpnelson replied:
good point

jpnelson committed:
Fix multiple calls of done in tests

The problem was that done() was called twice, since we returned a stream. When this stream "ended", it considered the task done. We then manually called done() too.

However, we can't just return the stream, as "end" of the stream is too early (it's not after the server completes tests). So we needed to not return the stream, and call done() manually.

Finally, we needed to consume the stream (as gulp does in orchestrator/lib/runTask.js) in order to "ensure that the stream completes". Otherwise the stream is not run at all.

jpnelson committed:
Respond to PR comments: resolve

joscha commented:
LGTM

treshugart commented:
lgtm

jpnelson commented:
@treshugart can you merge this? Looks like the lgtm hook is broken
